src/Apps/Artwork/ArtworkApp.tsx

```diff
@@ -87,6 +89,15 @@ const BelowTheFoldArtworkDetails: React.FC<BelowTheFoldArtworkDetailsProps> = ({
 export const ArtworkApp: React.FC<Props> = props => {
   const { artwork, me, referrer, tracking, shouldTrackPageView } = props
   const { match } = useRouter()
+  const hasArtists = (artwork.artists?.length ?? 0) > 0
+  const biddingEndAt =
+    artwork?.saleArtwork?.extendedBiddingEndAt ?? artwork?.saleArtwork?.endAt
+  const { hasEnded } = useTimer(biddingEndAt!, artwork?.sale?.startAt!)
+
+  // TODO: a mix of logic from displaying a "Bidding closed" message from
+  // ArtworkSidebar/ArtworkSidebar and our assumptions
+  const displayAuctionCreateAlertHeader =
+    hasArtists && (hasEnded || lotIsClosed(artwork.sale, artwork.saleArtwork))
```

Comment:
@gkartalis We tried to use the existing logic as in ArtworkSidebar since it's the same functionality, let us know if there is a way to improve this.